packages/devtools_app/lib/src/shared/preferences.dart

```diff
@@ -326,7 +326,7 @@ class MemoryPreferencesController extends DisposableController
   /// when browsing references in console.
   final refLimitTitle = 'Limit for number of requested live instances.';
   final refLimit = ValueNotifier<int>(_defaultRefLimit);
-  static const _defaultRefLimit = 100;
+  static const _defaultRefLimit = 100000;
```

Comment:
Does this correspond to a maximum number of VM service objects that can be requested / tracked? If so, there's a good chance service IDs will expire before you hit this limit.

Reply:
It is parameter, passed to getInstances at the moment. It seems we will not need it as soon as we will start using getInstancesAsArray.